files/default/ec2_dev_2_volid.py

```diff
@@ -4,24 +4,26 @@
 import urllib2
 import sys
 import os
+import syslog
+import time
 
 # Get dev
 try:
-  dev = str(sys.argv[1])
+    dev = str(sys.argv[1])
 except IndexError:
-  print "Provide block device i.e. xvdf"
-  sys.exit(1)
+    syslog.syslog(syslog.LOG_ERR, "%s - Provide block device i.e. xvdf" % os.path.basename(__file__))
```

Comment:
(minor) since syslog.sysog call is used multiple times and follows a format could you provide a function for it? the caller should just pass the message to append

Reply:
syslog.syslog was already logging the script name, so no format is needed. Only the message is passed to the function